<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图书管理系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1351</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>单本入库</string>
     </attribute>
     <widget class="QTextBrowser" name="tb_1">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>210</y>
        <width>671</width>
        <height>271</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>27</x>
        <y>19</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>书号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="bno">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>17</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="catgr">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>17</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>类目</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="title">
      <property name="geometry">
       <rect>
        <x>524</x>
        <y>17</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>474</x>
        <y>20</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>标题</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="press">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>57</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>27</x>
        <y>60</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>出版社</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="year">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>57</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>年份</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>100</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>标题</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="author">
      <property name="geometry">
       <rect>
        <x>524</x>
        <y>57</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>474</x>
        <y>60</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>作者</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ttl">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>98</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>27</x>
        <y>101</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>总量</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>101</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>库存</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="stock">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>98</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="check_in">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>150</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>入库</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clear_1">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>150</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>474</x>
        <y>101</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>价格</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="price">
      <property name="geometry">
       <rect>
        <x>524</x>
        <y>98</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="clear_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>150</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>清空记录</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>批量入库</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>选择文件</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>671</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>入库</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>671</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>查询</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>671</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>精确查找</string>
      </property>
      <widget class="QPushButton" name="bno_btn">
       <property name="geometry">
        <rect>
         <x>503</x>
         <y>23</y>
         <width>112</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>查找</string>
       </property>
      </widget>
      <widget class="QLabel" name="b1">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>101</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>按书号查找</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="bno_ser">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>27</y>
         <width>331</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="bno_btn_2">
       <property name="geometry">
        <rect>
         <x>503</x>
         <y>63</y>
         <width>112</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>查找</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="title_ser">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>67</y>
         <width>331</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="b1_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>按书名查找</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>671</width>
        <height>341</height>
       </rect>
      </property>
      <property name="title">
       <string>范围查找</string>
      </property>
      <widget class="QLabel" name="b1_3">
       <property name="geometry">
        <rect>
         <x>91</x>
         <y>75</y>
         <width>41</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>类目</string>
       </property>
      </widget>
      <widget class="QLabel" name="b1_4">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>113</y>
         <width>61</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>出版社</string>
       </property>
      </widget>
      <widget class="QLabel" name="b1_5">
       <property name="geometry">
        <rect>
         <x>91</x>
         <y>153</y>
         <width>41</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>年份</string>
       </property>
      </widget>
      <widget class="QLabel" name="b1_6">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>193</y>
         <width>41</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>作者</string>
       </property>
      </widget>
      <widget class="QLabel" name="b1_7">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>234</y>
         <width>41</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>价格</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ser_cat">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>431</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ser_aut">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>190</y>
         <width>431</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ser_year_1">
       <property name="geometry">
        <rect>
         <x>141</x>
         <y>150</y>
         <width>181</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ser_year_2">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>150</y>
         <width>191</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>154</y>
         <width>81</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>到</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ser_pri_2">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>230</y>
         <width>191</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ser_pri_1">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>230</y>
         <width>181</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>234</y>
         <width>81</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>到</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ser_pre">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>109</y>
         <width>411</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="b1_8">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>34</y>
         <width>41</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>书名</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ser_bno">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>431</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>280</y>
         <width>112</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>查找</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>借书/还书</string>
     </attribute>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>24</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>卡号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cno_chk">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>581</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>64</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>书号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="bno_chk">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>581</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>借书</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>100</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>还书</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="tb_chk">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>671</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>100</y>
        <width>221</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>查询该卡借阅信息</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <property name="toolTip">
      <string/>
     </property>
     <attribute name="title">
      <string>借书证管理</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>671</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>删除借书证</string>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>36</y>
         <width>81</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>卡号</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="cno">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>32</y>
         <width>580</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>76</y>
         <width>112</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>76</y>
         <width>112</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>671</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>添加/修改借书证</string>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>32</y>
         <width>81</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>卡号</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="cno_add">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>140</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="name">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>30</y>
         <width>130</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>32</y>
         <width>81</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>姓名</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="dept_add">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>68</y>
         <width>571</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>81</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>部门</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>32</y>
         <width>81</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>类别</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>69</x>
         <y>105</y>
         <width>112</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>添加/修改</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>30</y>
         <width>99</width>
         <height>24</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>教师</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>学生</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QTextBrowser" name="card_text">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>671</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>713</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
